documentation/tutorial.md

```diff
@@ -159,54 +157,85 @@ please hold on. We're going to cover it in the next section.
 The best way to learn how to extract data with Crawly is trying the
 selectors in Crawly shell.
 
-1. Start the Elixir shell using `iex -S mix` command
+1. Start the Elixir shell using `iex -S mix` command.
 2. Now you can fetch a given HTTP response using the following
    command:
    `response = Crawly.fetch("https://www.homebase.co.uk/our-range/tools")`
 
 You will see something like:
 
 ```
-{:ok,
- %HTTPoison.Response{
-   body: "[response body here...]"
-   headers: [
-     {"Date", "Fri, 24 May 2019 02:37:26 GMT"},
-     {"Content-Type", "text/html; charset=utf-8"},
-     {"Transfer-Encoding", "chunked"},
-     {"Connection", "keep-alive"},
-     {"Cache-Control", "no-cache, no-store"},
-     {"Pragma", "no-cache"},
-     {"Expires", "-1"},
-     {"Vary", "Accept-Encoding"},
-     {"Set-Cookie", "Bunnings.Device=default; path=/"},
-     {"Set-Cookie",
-      "ASP.NET_SessionId=bcb2deqlapednir0lysulo1h; path=/; HttpOnly"},
-     {"Set-Cookie", "Bunnings.Device=default; path=/"},
-     {"Set-Cookie",
-      "ASP.NET_SessionId=bcb2deqlapednir0lysulo1h; path=/; HttpOnly"},
-     {"Set-Cookie", "Bunnings.UserType=RetailUser; path=/"},
-     ....,
-     {"Set-Cookie",
-      "__AntiXsrfToken=fd198cd78d1b4826ba00c24c3af1ec56; path=/; HttpOnly"},
-     {"Server", "cloudflare"},
-     {"CF-RAY", "4dbbe33fae7e8b20-KBP"}
-   ],
-   request: %HTTPoison.Request{
-     body: "",
-     headers: [],
-     method: :get,
-     options: [],
-     params: %{},
-     url: "https://www.homebase.co.uk/our-range/tools"
-   },
-   request_url: "https://www.homebase.co.uk/our-range/tools",
-   status_code: 200
- }}
+%HTTPoison.Response{
+  body: "[response body here...]"
+  headers: [
+    {"Date", "Fri, 17 Apr 2020 10:34:35 GMT"},
+    {"Content-Type", "text/html; charset=utf-8"},
+    {"Transfer-Encoding", "chunked"},
+    {"Connection", "keep-alive"},
+    {"Set-Cookie",
```

Comment:
I am not sure we need all these Set-Cookie everywhere. Please remove some of them, so the tutorial is shorter

Reply:
Sure, I will do that.